shared/scripts/prepare.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Let's use a node script for this instead, so that it's more portable.

You can see an example of one in https://github.com/aragon/aragon/blob/master/scripts/start.

Reply:
done